spec/lib/zoom/actions/user/token_spec.rb

```diff
@@ -4,21 +4,32 @@
 
 describe Zoom::Actions::User do
   let(:zc) { zoom_client }
-  let(:args) { { id: 'ufR9342pRyf8ePFN92dttQ' } }
+  let(:args) { { user_id: 'ufR9342pRyf8ePFN92dttQ' } }
 
   describe '#user_token action' do
     context 'with a valid response' do
       before :each do
         stub_request(
           :get,
-          zoom_url("/users/#{args[:id]}/token")
+          zoom_url("/users/#{args[:user_id]}/token")
         ).to_return(status: 200,
                     body: json_response('user', 'token'),
                     headers: { 'Content-Type' => 'application/json' })
       end
 
-      it 'requires id param' do
-        expect { zc.user_token(filter_key(args, :id)) }.to raise_error(Zoom::ParameterMissing, '[:id]')
+      it 'requires user_id param' do
+        expect { zc.user_token(filter_key(args, :user_id)) }
+          .to raise_error(Zoom::ParameterMissing, '[:user_id]')
+      end
+
+      it 'allowes type' do
+        args[:type] = 'login'
```

Comment:
This is go align with the API docs
```suggestion
        args[:type] = 'token'
```